libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/TabsView.tsx

```diff
@@ -180,6 +186,19 @@ export class TabsView extends React.PureComponent<
                     />
                   </>
                 )}
+              {t.key === GlobalTabKeys.ForecastingTab && (
+                <>
+                  <h3
```

Comment:
we try to move away from browser default h1,h2...
is <Label> from fluent ui good enough?

Reply:
I just copied what the other tabs are doing, e.g., Error Analysis right below. What would you prefer? `<Text variant="xLarge">...</Text>` ? I can change all other occurrences, too.

Reply:
<Text variant="xLarge"> is good.
EA could be caused by legacy code.

Reply:
Sounds good. I'll create a separate PR since there are quite a few of them.